### PR TITLE
Upgrade module from template

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Changed
+
+- Updated to new module structure
```
Comment (waxb) [inline CHANGELOG.md:12]: i think this can be omitted

File: test/terramate_aws.tm.hcl [hunk 2]
```diff
@@ -1,45 +1,41 @@
-# globals {
```
Comment (waxb) [inline test/terramate_aws.tm.hcl:1]: this file can be omitted

**Review by waxb — COMMENTED** [review]
terramate manifest is missing

File: test/terramate_aws.tm.hcl (hunk 2 above)
Comment (soerenmartius) [reply test/terramate_aws.tm.hcl:1]: good catch 🦅 resolved

File: CHANGELOG.md (hunk 1 above)
Comment (soerenmartius) [reply CHANGELOG.md]: resolved

**Comment by soerenmartius** [comment]
> terramate manifest is missing

my bad, added the manifest also!

**Review by zied-elouaer — APPROVED** [review]
LGTM

**Review by waxb — APPROVED** [review]
lgtm

**Pull request merged**